chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+
+---
+
+## Beef is highly inefficient use of land to generate calories
+
+```{python}
+# TODO save wrangled data
+land_use = pd.read_csv('data/land-use-kcal-poore.csv').rename(columns={'Land use per 1000kcal (Poore & Nemecek, 2018)': 'Land use per 1000kcal (m2)'}).query('Entity in ["Barley", "Maize", "Rice", "Tofu (soybeans)", "Cassava", "Bananas", "Eggs", "Poultry Meat", "Dark Chocolate", "Milk", "Cheese", "Coffee", "Lamb & Mutton", "Beef (beef herd)"]')
+alt.Chart(land_use).mark_bar().encode(
+    alt.X('Land use per 1000kcal (m2)'),
+    alt.Y('Entity', sort='x', title=''))
+```
+
+Notes:
+The plot in this slide reveals that herding cattle for the purpose of beef production
+is one of the most inefficient ways to produce food for human consumption.
+
+Together with the large demand for beef products
+this inefficient use of land creates an unfortunate situation in Brazil
+where a lot of tropical forest is being cut down to make room for new pastures.
+
+When telling a story,
+it is important to not jump to conclusions
+or suggest that solutions to problems are simpler than what they might really be.
+
+So far we have seen that beef is a major contributing factor to deforestation in Brazil,
+but there are many more aspect of the food production chain that we have not considered,
+such as the use of other resources
+
+Changing beef for other products would be highly beneficial,
+but how achievable it is 
+and what cost is associated with making the change
+we don't know from the data we have explored here.
+
+Let's continue our story by looking at the same breakdown for Indonesia,
+is beef production the biggest issue there as well?
+
+---
+
+## Palm oil is the biggest contributor to deforestation in Indonesia
+
+```{python}
+# TODO save wrangled data
+indonesia_drivers = pd.read_csv('data/deforestation-drivers-indonesia.csv').query('2013 > Year > 2003').groupby('Entity').sum().reset_index()
+alt.Chart(indonesia_drivers, title='Palm oil was the leading cause of deforestation between 2003 and 2013 in Indonesia').mark_bar().encode(
+    x='deforestation_hectares',
+    y=alt.Y('Entity', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Indonesia
+between the years 2003 and 2013.
+
+Here,
+it is palm oil that drives most of the deforestation,
+although small scale clearings, timber plantation, and grasslands also contribute substantially.
+You might have read many negative things about palm oil already,
+as it has been in media's focus due to its link to tropical deforestation.
+
+From this chart alone it surely seems that decreasing palm oil production
+would lead to less deforestation.
+
+But if palm oil production would decrease
+while the world's demand for vegetable oils remains constant,
+another oil crop would need to take its place.
+
+Similar to what we did for beef,
+let's explore how efficient palm oil and its alternatives are
+when it comes to land use versus yield.
```

Comment:
```suggestion
Here, we see that most of the deforested land in Indonesia is used for palm oil
plantations. We see that much deforested land is also used for small scale 
clearings, timber plantations, and grasslands.

Perhaps you have have read, or heard, negative things about palm oil in the past? 
It has been of focus of late in the media, due to its link to tropical deforestation.

From this chart alone, we might want to recommend that decreasing palm oil production
might be a good intervention to reduce deforestation in Indonesia.

However, if this were to be done, we would need another oil crop would need to take its place 
to keep up with the world's demand for vegetable oils.

Let's explore how efficient palm oil and its alternatives are
when it comes to land use versus yield.
```

Reply:
Note - we've here somehow dropped the narrative/link that global demand for exports leads to deforestation, is palm oil one of Indonesia's big exports? We have a similar gap in the beef story. Perhaps it is there but it might need to be fleshed out?

Reply:
We don't have data on how much of each product is exported, that is why I don't talk about it and phrased this part as a second story. I think we can add a sentence to make that clear, partiuclarly now that we are explicitly continuing the same story.

Reply:
I agree we should make it explicit we do not have such data, so the link is not there for sure.

Reply:
I added this to the brazil section already and will mention it here again.